File: app/lib/models/pet/pets_view_model.dart
class PetsViewModel {
  late int id;
  late String name;
  late String brand;
  late String createdAt;

  PetsViewModel(
      {required this.id,
      required this.name,
      required this.brand,
      required this.createdAt});

  PetsViewModel.fromJson(Map<String, dynamic> json) {
    id = json["id"];
    name = json["name"];
    brand = json["brand"];
    createdAt = json["createdAt"];
  }
}

File: server/src/main/java/com/alp/PetAcademy/models/pet/response/PetsViewModel.java
package com.alp.PetAcademy.models.pet.response;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class PetsViewModel {
    private int id;
    //TODO: Return image
    private String name;
    private String brand;
    private String createdAt;
}
